views/web.py

```diff
@@ -286,6 +280,7 @@ def love():
             flash('Secret love sent to {}!'.format(real_display_str))
             return redirect(url_for('home'))
         else:
+            flash('Love sent to {}!'.format(real_display_str))
```

Comment:
this shouldn't be done here. /sent doesn't consume flash messages (we know that if I user gets to the /sent endpoint, they have successfully sent a message and we tell them that directly on the template). Right now, this flash will get consumed the next time a user hits the homepage which is *super* confusing.